gatewayconfig/bluetooth/characteristics/wifi_connect_characteristic.py

```diff
@@ -98,6 +114,16 @@ def WriteValue(self, value, options):
         self.wifi_service = str(wifi_details.service)
         self.wifi_password = str(wifi_details.password)
 
+        if self.wifi_service and self.wifi_password:
```

Comment:
don't understand what is the purpose of this thread, after setting ssid and password on network manager and setting it to autoconnect, network manager is supposed to take care of retrying reasonably.

Reply:
@pritamghanghas If autoconnect feature of nmci is used, is there a way to hook up the finish event? Because the maker app need to notify about the correctness of the SSID/Password to the user.

Reply:
@posterzh no that will only be possible with dbus and evenloop. we can keep this the way it is for now.